ephypype/spectral.py

```diff
@@ -3,13 +3,15 @@
 Author: David Meunier <david_meunier_79@hotmail.fr>
 """
 import os
-
 import numpy as np
+
 from scipy.io import savemat
 
 from mne.connectivity import spectral_connectivity
 from mne.viz import circular_layout, plot_connectivity_circle
 
+import matplotlib.pyplot as plt
```

Comment:
can you make the matplotlib import nested? Otherwise it will result in weird bugs ...

Reply:
> can you make the matplotlib import nested? Otherwise it will result in weird bugs ...

done